src/api/goods.ts

```diff
@@ -10,10 +10,16 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
```

Comment:
here you modify entered data array, not sure that its correct, you can try this :
`const goods = await getAll();

  return [...goods]
    .sort((good1, good2) => good1.name.localeCompare(good2.name))
    .splice(0, 5);`